src/main/java/seedu/duke/command/GetCommand.java

```diff
@@ -0,0 +1,23 @@
+package seedu.duke.command;
+
+import seedu.duke.model.Item;
+import seedu.duke.model.ItemContainer;
+
+public class GetCommand extends Command {
+
+    private final Item selectedItem;
+
+    public GetCommand(String name, ItemContainer list) {
+        this.selectedItem = list.getItem(name);
+    }
+
+    /**
+     * Executes the operation of adding the item to the list.
```

Comment:
Isn't this GetCommand instead of adding command

Reply:
Good catch, amended